packages/swayze/CHANGELOG.md

```diff
@@ -1,14 +1,22 @@
-# 1.2.0
+# 1.4.0
 
 - Deprecates `id` property on `UserSelectionModel`.
 
+# 1.3.0
+
+- Allow resizing of columns or rows.
+
+# 1.2.0
+
+- Add drag and drop for columns and rows.
+
```

Comment:
Isn't this part of the same release? Can't we aggregate all these changes inside 1.2.0, since they haven't been released to pub.dev yet?

Reply:
sounds good